scripts/mdpow-solvationenergy

```diff
@@ -185,31 +226,48 @@ if __name__ == "__main__":
     parser.add_option('-p', '--plotfile', dest="plotfile",
                       help="plot dV/dlambda to FILE; use png or pdf suffix to "
                       "determine the file type. 'auto' generates a pdf file "
-                      "DIRECTORY/dVdl_<molname>_<solvent>.pdf and 'None' disables it [%default]",
+                      "DIRECTORY/dVdl_MOLID_pow.pdf and 'None' disables it [%default]"
+                      "The plot function is only available for mdpow estimator,"
+                      "and is disabled when using alchemlyb estimators.",
+                      metavar="FILE")
+    parser.add_option('--solvent', '-S', dest='solvent',
+                      metavar="NAME",
+                      help="solvent NAME for compound, 'water', 'octanol', or 'cyclohexane' [%default]")
```

Comment:
Will this option take other water solvents like "tip3p" or do we need to do more so that we can work with different water solvents as for #145 ?

Reply:
We need to change the directory names used in `mdpow.equilibrium` and the corresponding setup scripts. Just change this option would not support different water solvents.

Reply:
ok, then that's going to be a bigger lift; thanks for the explanation